python/python/glide/async_commands/cluster_commands.py

```diff
@@ -579,36 +586,41 @@ async def sort_store(
             int: The number of elements in the sorted key stored at `store`.
 
         Examples:
-            >>> await client.lpush("mylist", 3, 1, 2)
-            >>> await client.sort_store("mylist", "sorted_list")
+            >>> await client.lpush(b"mylist", b'3', b'1', b'2')
```

Comment:
```suggestion
            >>> await client.lpush(b"mylist", [b'3', b'1', b'2'])
```
this is the correct syntax